spec.html

```diff
@@ -9543,10 +9542,10 @@
         &lt;ZWJ&gt;
 
       UnicodeIDStart ::
-        &gt; any Unicode code point with the Unicode property &ldquo;ID_Start&rdquo;
+        &gt; any Unicode code point with the Unicode property “ID_Start”
```

Comment:
These would be the first non-ASCII symbols in `spec.html`. Are we sure this doesn’t break anyone’s scripts? (It shouldn’t, but hey…)

FWIW, here’s the ecmarkup bug: https://github.com/bterlson/ecmarkup/issues/118. Fixing that would be preferable to this workaround IMHO.